### PR TITLE
Add meeting notes from July

### PR DESCRIPTION
**Pull request by egeakman**
----
:books: Documentation preview :books:: https://docs-community--84.org.readthedocs.build/en/84/

### CODE REVIEW

File: docs/monthly-meeting/2023-07.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Documentation Community Team Meeting (July 2023)
+
+- **Date:** 2023-07-03
+- **Time:** [19:00 UTC](https://arewemeetingyet.com/UTC/2023-07-03/19:00/Docs%20Meeting) 
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
```
Comment (hugovk) [inline docs/monthly-meeting/2023-07.md:5]: Let's link it:
```suggestion
- **This HackMD:** [https://hackmd.io/@encukou/pydocswg1](https://hackmd.io/@encukou/pydocswg1)
```

File: docs/monthly-meeting/2023-07.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* Programmatic handling of `deprecated-removed`
+    * Followup from [soft deprecation discussion](https://discuss.python.org/t/formalize-the-concept-of-soft-deprecation-dont-schedule-removal-in-pep-387-backwards-compatibility-policy/27957/73)
+    * MVP should be up by next meeting:
+        * A list of everything that's deprecated
+        * docs on the directive in the devguide
+
+* Thoughts on putting a translation progress notice in docs pages (for each language)? [python/docsbuild-scripts#148](https://github.com/python/docsbuild-scripts/issues/148)
+    * Sounds like a good idea
+    * Sphinx issue is open
+        * This should be done in Sphinx first - Manuel will look into it (https://github.com/sphinx-doc/sphinx/issues/11157)
```
Comment (hugovk) [inline docs/monthly-meeting/2023-07.md:45]: I've made a CLI to help with these links!

```console
$ python -m pip install -U linkotron
$ linky --md https://github.com/sphinx-doc/sphinx/issues/11157
Copied! [sphinx-doc/sphinx#11157](https://github.com/sphinx-doc/sphinx/issues/11157)
```


```suggestion
        * This should be done in Sphinx first - Manuel will look into it ([sphinx-doc/sphinx#11157](https://github.com/sphinx-doc/sphinx/issues/11157))
```

File: docs/monthly-meeting/2023-07.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Ned / `@nedbat`
+- Manuel / `@humitos`
+
+
+## Discussion
+
+* Thoughts on MDN icons (see e.g. the ToC sidebar for [Document](https://developer.mozilla.org/en-US/docs/Web/API/Document)), as a compact way to flag API status in overviews?
+    * :wastebasket: = deprecated
+    * :warning: = non-standard/implementation-specific
+    * 🧪 = experimental/unstable
```
Comment (hugovk) [inline docs/monthly-meeting/2023-07.md:34]:
```suggestion
    * 🗑️ = deprecated
    * ⚠️ = non-standard/implementation-specific
    * 🧪 = experimental/unstable
```

**Review by hugovk — CHANGES_REQUESTED** [review]

**Comment by egeakman** [comment]
Looks like I'm a little out of focus today

**Review by hugovk — APPROVED** [review]
I think we're all tired after EuroPython and the sprints are quite hectic too!

Thank you for the PRs!

**Pull request merged**